pkg/proc/variables.go

```diff
@@ -558,6 +560,27 @@ func (v *Variable) parseG() (*G, error) {
 
 	status, _ := constant.Int64Val(v.fieldVariable("atomicstatus").Value)
 	f, l, fn := v.bi.PCToLine(uint64(pc))
+
```

Comment:
GetG, and consequently parseG, is called quite a lot. I would prefer if parseG only saved the value of labelsVar and then a method of *G did the rest of the work and cached the result.

Reply:
Does it make sense to reuse the variable field and call `loadFieldNamed("labels")` in the method of *G?

Reply:
Yes, that's fine.